Makefile

```diff
@@ -24,7 +24,7 @@ scripts/protocc/protocc.py: scripts/protocc
 	git submodule update
 
 proto: scripts/protocc/protocc.py
-	python3 scripts/protocc/protocc.py --out go
+	python3 scripts/protocc/protocc.py --out go && toast proto
```

Comment:
this isn't right. right now, this makefile rule only depends on scripts/protocc/protocc.py.

can you create a new makefile rule with the correct dependencies?